GPU/Vulkan/DrawEngineVulkan.cpp

```diff
@@ -588,17 +588,10 @@ void DrawEngineVulkan::DoFlush() {
 
 	VulkanRenderManager *renderManager = (VulkanRenderManager *)draw_->GetNativeObject(Draw::NativeObject::RENDER_MANAGER);
 	
-	// TODO: Move this into the below if statement. Little scary change though so holding off until 1.11.
+	// TODO: Needs to be behind a check for changed render pass, at an appropriate time in this function.
+	// Similar issues as with the lastRenderStepId_ check. Will need a bit of a rethink.
```

Comment:
In theory, a shader blend shouldn't actually cause any new state - it might cause a new render pass, motivating viewport/scissor/stencil to be reapplied, but it won't actually change the values.

So I'd think the pipeline would - in theory - not be affected by that?  And if anything is dirty causing us to shader blend again, then that would prevent `lastPipeline_` from being used, right?

-[Unknown]